outputs.tf

```diff
@@ -1,89 +1,55 @@
-// Output the IDs of the ECS instances created
-output "vpc_id" {
-  description = "Deprecated and use this_vpc_id instead"
-  value       = local.this_vpc_id
-}
-
-output "cidr_block" {
-  description = "Deprecated and use this_vpc_cidr_block instead"
-  value       = concat(alicloud_vpc.vpc.*.cidr_block, [""])[0]
-}
 
-output "vswitch_ids" {
-  description = "Deprecated and use this_vswitch_ids instead"
-  value       = alicloud_vswitch.vswitches.*.id
-}
-
-output "availability_zones" {
-  description = "Deprecated and use this_availability_zones instead"
-  value       = alicloud_vswitch.vswitches.*.availability_zone
-}
-
-output "router_id" {
-  description = "Deprecated and use this_router_id instead"
-  value       = alicloud_route_entry.route_entry.*.router_id
-}
-
-output "route_table_id" {
-  description = "Deprecated and use this_route_table_id instead"
-  value       = alicloud_route_entry.route_entry.*.route_table_id
+output "vpc_id" {
+  description = "The VPC ID"
+  value       = local.vpc_id
 }
 
-output "this_vpc_id" {
-  description = "The VPC id"
-  value       = local.this_vpc_id
+output "vpc_cidr_block" {
```

Comment:
Please improve the outputs and set a prefix `this_` for them.

Reply:
I am a bit lost on the advantage to use "this". When I saw other Terraform modules in GitHub, the maintainer omits the name of the resource, in this case "_this_" because it is unique.

You can take as example these other alibaba modules:

- https://github.com/terraform-alicloud-modules/terraform-alicloud-kubernetes/blob/master/outputs.tf

I propose I add the "this" and I keep also the other ones.

As for the description, the "The VPC id" is the actual description of today's module. How do you suggest to improve ?
"The VPC id created by the Terraform module" ?

Reply:
I have not worked on the alicloud kubernetes, no PRs from my side at the moment. I looked at other official terraform modules and "this" or other identifiers are not used when there is only one type of resource.
I think we should follow this current best practice as it is also easier to read

In any case, I removed the duplicates ,)